sql/amo_prod/desktop_addons_by_client_v1/metadata.yaml

```
@@ -0,0 +1,15 @@
+friendly_name: Desktop addons by client
+description: >
```

Nit: To prevent trialing newlines in case this description shows up in BigQuery
```suggestion
description: >-
```